x.ps1

```diff
@@ -24,5 +24,12 @@ foreach ($python in "py", "python3", "python", "python2") {
     }
 }
 
+$found = (Get-Command 'python*' -CommandType Application -ErrorAction SilentlyContinue | Where-Object {$_.name -match 'python\d'})
```

Comment:
This `Get-Command` is different from the one above (it includes `-CommandType Application`). What does that do? Does it make sense to have that check above as well? If so, let's extract this into a function so they stay in sync.

Reply:
This allows to only check for binaries, and ignore cmdlets, aliases, etc...
It should also be applied above, because otherwise it may fail.

Reply:
Great, thanks. Please add that function I mentioned above.